internal/worker/json.go

```diff
@@ -94,6 +94,14 @@ type DepsolveJobResult struct {
 	ErrorType    ErrorType                      `json:"error_type"`
 }
 
+type DataJob struct {
```

Comment:
I would honestly go rather with specialized jobs rather than one generic one. For this case, I would just name it ManifestJob. It wouldn't have any arg fields, and just have `Manifest` as the only field of the result.

Reply:
Agreed, would you make the type only dequeue-able by ID? As in the worker server would throw an error when you try to dequeue this job by type?

Reply:
Makes sense!